src/azure-firewall/azext_firewall/_params.py

```diff
@@ -149,12 +149,12 @@ def load_arguments(self, _):
 
     with self.argument_context('network firewall policy rule-collection-group') as c:
         c.argument('firewall_policy_name', options_list=['--policy-name'], help='The name of the Firewall Policy.')
-        c.argument('rule_group_name', options_list=['--name', '-n'], help='The name of the Firewall Policy Rule Collection Group.')
+        c.argument('rule_collection_group_name', options_list=['--name', '-n'], help='The name of the Firewall Policy Rule Collection Group.')
         c.argument('priority', type=int, help='Priority of the Firewall Policy Rule Collection Group')
 
     with self.argument_context('network firewall policy rule-collection-group collection') as c:
-        c.argument('rule_group_name', options_list=['--rule-collection-group-name'], help='The name of the Firewall Policy Rule Collection Group.')
-        c.argument('rule_name', options_list=['--name', '-n'], help='The name of the collection in Firewall Policy Rule Collection Group.')
+        c.argument('rule_collection_group_name', options_list=['--rule-collection-group-name'], help='The name of the Firewall Policy Rule Collection Group.')
```

Comment:
why change?

Reply:
I didn't change the UX for customers. `rule_group_name` is the old name. `rule_collection_group_name` is the new name.